framework/include/base/SubProblem.h

```diff
@@ -368,6 +376,10 @@ class SubProblem : public Problem
   /* This needs to remain <unsigned int> for threading purposes */
   std::vector<unsigned int> _has_active_elemental_moose_variables;
 
+  std::vector<std::set<unsigned int> > _active_material_property_ids;
```

Comment:
We are starting to close the space in `> >` now. Feel free to do that in new code you commit.